app.js

```diff
@@ -69,8 +69,9 @@ app.use(function(err, req, res, next) {
   res.status(err.status || 500);
   res.render('error', {
     message: err.message,
-    error: {}
+    error: {},
   });
 });
 
+
```

Comment:
We should use a single blank line between functions definitions. Adding multiple blank lines is permitted but not required or encouraged. Would you mind removing the extra blank lines? Unless you have a different point of view on this matter.